src/Sentry/PlatformAbstractions/RuntimeInfo.cs

```diff
@@ -83,7 +83,7 @@ internal static void SetNetCoreVersion(Runtime runtime)
             {
                 // https://github.com/dotnet/BenchmarkDotNet/issues/448#issuecomment-308424100
                 var assembly = typeof(System.Runtime.GCSettings).GetTypeInfo().Assembly;
-#if NET5_0
+#if NET5_0 || NETCOREAPP3_0
```

Comment:
if not now, when net6 lands and we come to this line again, it'll make more sense to flip: `if !NETFX && !NETSTANDARD` :)